### ISSUE TITLE
Crash in NetworkConfigLoader.initNetworkConfig

### ISSUE THREAD

**Issue by biafra23** [unstoppabledomains/resolution-java#37]
We integrated this library and it crashes on us :-( 

Steps to reproduce (on production mainnet and rinkeby):
- Add safe
- Select "unstoppbable domain"
- Observe Crash

Expected bahviour: No crash. 

See: https://github.com/gnosis/safe-android/issues/1398 for more stack traces. This seems to be the root cause:

```
Caused by com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "version" (class com.unstoppabledomains.config.network.model.NetworkConfig), not marked as ignorable (one known property: "networks"])
 at [Source: (String)"{
    "version": "1.1.0",
    "networks": {
        "1": {
            "contracts": {
                "Registry": {
                    "address": "0xD1E5b0FF1287aA9f9A268759062E4Ab08b9Dacbe",
                    "legacyAddresses": []
                },
                "SignatureController": {
                    "address": "0x82EF94294C95aD0930055f31e53A34509227c5f7",
                    "legacyAddresses": []
                },
                "MintingController": {
                    "address"[truncated 3440 chars]; line: 2, column: 17] (through reference chain: com.unstoppabledomains.config.network.model.NetworkConfig["version"])
       at com.fasterxml.jackson.databind.DeserializationContext.handleUnknownProperty(DeserializationContext.java:855)
       at com.fasterxml.jackson.databind.deser.std.StdDeserializer.handleUnknownProperty(StdDeserializer.java:1212)
       at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownProperty(BeanDeserializerBase.java:1604)
       at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownVanilla(BeanDeserializerBase.java:1582)
       at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:299)
       at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:156)
       at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4526)
       at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3468)
       at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3436)
       at com.unstoppabledomains.config.network.NetworkConfigLoader.initNetworkConfig(NetworkConfigLoader.java:43)
       at com.unstoppabledomains.config.network.NetworkConfigLoader.<clinit>(NetworkConfigLoader.java:19)
       at com.unstoppabledomains.config.network.NetworkConfigLoader.getContractAddress(NetworkConfigLoader.java:26)
       at com.unstoppabledomains.resolution.naming.service.CNS.<init>(CNS.java:32)
       at com.unstoppabledomains.resolution.Resolution$Builder$2.<init>(Resolution.java:283)
       at com.unstoppabledomains.resolution.Resolution$Builder.build(Resolution.java:282)
```
This affects several Android versions and several manufacturers.
Please let me know if you need more details about these crashes.

**Comment by biafra23** [comment]
Here is a possible workaround.

Add 

`@JsonIgnoreProperties(ignoreUnknown = true)`
to com.unstoppabledomains.config.network.model.Contract and com.unstoppabledomains.config.network.model.NetworkConfig

Adding a setter doesn't work. 

Removing the properties (version and legacyAddresses ) from the json file also works.

**Comment by JohnnyJumper** [comment]
Thank you, so much for letting us know about this. We are working on the issue.

**Comment by JohnnyJumper** [comment]
Should be fixed by #39

**Comment by JohnnyJumper** [comment]
Looks like the problem is still persist. Working on a fix
